views/personal/address.pug

```diff
@@ -26,13 +26,6 @@ block content
               div #{data.personal.address.city}, #{data.personal.address.province}
               div #{data.personal.address.postalCode}
 
-  div
-    details.
-      <summary><span>Help with mailing address</span></summary>
```

Comment:
Why are we removing this?

Reply:
I asked @brianhendrick1 about it and he said that we don't have content for most of those, so to remove them.